src/starkware/signature.js

```diff
@@ -96,6 +96,16 @@ function assertInRange(input, lowerBound, upperBound, inputName = '') {
  points defined in the documentation.
 */
 export function pedersen(input) {
+  if (useCryptoCpp) {
+    if (typeof input[0] === 'string') {
+      input[0] = BigInt(`0x${input[0]}`);
+    }
+    if (typeof input[1] === 'string') {
+      input[1] = BigInt(`0x${input[1]}`);
+    }
+    return pedersenCpp(input[0], input[1]).toString(16);
```

Comment:
To review :warning: Is it fine to assume that hexa strings will never start with `0x`? At least it's the case in tests

Reply:
Yes that's ok.